### ISSUE TITLE
Single first page option.

### ISSUE THREAD

**Issue by ojvaj1** [kodjodevf/mangayomi#167]
### Describe your suggested feature

A setting for singeling the first page. Helps with alignment when you read double page.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.

**Comment by sho052** [comment]
Agreed. TachiyomiJ2K already has this feature. Without it, the alignment can be off for a lot of manga, making the double page option rather useless.

**Comment by sho052** [comment]
Doesn't seem that it has been adressed yet... Is there a way to adjust double page alignment already?

**Comment by tu-annguyen** [comment]
I wanted to open an issue on this because I had problems with the last or first page of the chapter not showing up when reading in double page mode. Thankfully it seems to already be worked on!
